formats/detailed_guide/publisher/links.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "topics",
+    "lead_organisations"
+  ],
+  "properties": {
+    "related_guides": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "alternative_format_provider_organisation": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "related_mainstream_url": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "additional_related_mainstream_url": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "alternative_scotland_url": {
```

Comment:
After discussion with Tatiana earlier, these might be external URLs which won't be in the content store. Has this been confirmed one way or the other?